### PR TITLE
Feature/daves adds

### PR DESCRIPTION
**Pull request by StuartCowley**

### CODE REVIEW

File: books.md [hunk 1]
```diff
@@ -1,11 +1,17 @@
 ## Terry Pratchett books
 - Good Omens
-- The Colour of Magic
+- I Shall Wear Midnight
 - Nightwatch
-- The Fifth Elephant
+- The Bromeliad
```
Comment (StuartCowley) [inline books.md:5]: Good choices

File: books.md [hunk 2]
```diff
@@ -1,11 +1,17 @@
...
 - The ocean at the end of the lane
 - Smoke and mirrors
 - Neverwhere
-- American Gods
\ No newline at end of file
+- American Gods
+
+## Philip K Dick
+- Ubik
+- Do Androids Cream Of Electric Sheep
```
Comment (StuartCowley) [inline books.md]:
```suggestion
- Do Androids Dream Of Electric Sheep
```

File: movies.md [hunk 3]
```diff
@@ -10,4 +10,11 @@
...
-- Bourne Legacy
\ No newline at end of file
+- Bourne Legacy
+
+## Michelle Yeoh movies
+
+- Everything Everywhere All At Once
+- Sunshine
+- Crouching Tiger Hidden Dragon
+- Tomorrow Never Dies
```
Comment (StuartCowley) [inline movies.md]:
```suggestion
- Crouching Tiger Hidden Dragon
- Sunshine
- Tomorrow Never Dies
```

**Review by StuartCowley — COMMENTED** [review]
A few changes required

File: movies.md (hunk 3 above)
Comment (devarrowsmith) [reply movies.md]: Implemented this suggestion.

**Pull request merged**